src/features/ProtectTransaction/components/PoweredByNansen.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Can we possibly extract this out to a general "powered by" component since it's mostly copy/paste?

Reply:
Are we sure this is necessary? It's usually very styling specific by provider isn't it?

Reply:
I don't think so. In the case of ENS and Zapper, im pretty sure we've just done the icon, and then `Powered By ${name}` with some small spacing between them.